### PR TITLE
Pushing development to main

### PR DESCRIPTION
**Pull request by Fahi28**
Pushing all completed features to main branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,33 +31,27 @@ The Earthquake Monitor project continuously ingests real-time earthquake data fr
 1. **Docker + ECR for ETL Deployment:**
 Using Docker allows for consistent development and runtime environments. Pushing the ETL to Amazon ECR ensures seamless integration with AWS Lambda, improving deployment speed and scalability.
 
-2. **AWS Lambda for ETL Processing:**
-Lambda provides a serverless, cost-effective solution for running the ETL every minute, ensuring high availability and automatic scaling without server management overhead.
+2. **AWS Lambda:**
+Lambda provides a serverless, cost-effective solution for running the ETL, short-to-long term data storage processing and alert notification via SNS, ensuring high availability and automatic scaling without server management overhead.
 
-3. **EventBridge Scheduler:**
+1. **EventBridge Scheduler:**
```
Comment (jiuliangut) [inline README.md:37]: Are the bullet points from here onwards numbered correectly?

**Review by jiuliangut — CHANGES_REQUESTED** [review]
looks good - but i think the numbering needs checking?

File: README.md (hunk 1 above)
Comment (Fahi28) [reply README.md:37]: it looks weird in the markdown but fine in the preview.

**Review by jiuliangut — APPROVED** [review]

**Review by S1mpySloth — APPROVED** [review]
All pre-approved from development, happy for these features to be pushed to main.

**Review by qrafiq1 — APPROVED** [review]
Everything looks good, got some nice features about to deployed.

Time to deploy!

**Pull request merged**